NOTES.md

```diff
@@ -0,0 +1,36 @@
+# Things Paul Came Across
+
+## General Python housekeeping I could do later
+
+- *Proper package*. Make imports more normal by changing `t-html` to something legally importable.
+- *pyproject.toml*. Put dependencies in there, then put everything under `src/thtml`. This would mean moving the
+  PyScript demo code up to root.
+- *Type hints*. Spread more of these around, for those that use IDEs that can autocomplete and squiggle.
+- *More comments*. To help people walking through the code.
```

Comment:
this stage is more *R&D* than definitive ... so far this was meant to be a playground ... once all the things we want work the way we want I can cleanup and add comments all over the place